### ISSUE TITLE
Ship with new starter 

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#17727]
We need to take a new starter 
1. change admin password back to admin
2. Run the "Maintenance > Tools > delete old asset versions"
3. Run the "Maintenance > Tools > Clean Assets "

**Comment by jgambarios** [comment]
Also we need to execute the default actions tasks before to generate a new starter:

```
curl -v -u "admin@dotcms.com:admin" -X POST http://localhost:8080/api/v1/upgradetask/ --header "Content-Type:application/json" -d '{ 
    "upgradeTaskClass":"com.dotmarketing.startup.runonce.Task05175AssignDefaultActionsToTheSystemWorkflow"
}' | jq
```
```
curl -v -u "admin@dotcms.com:admin" -X POST http://localhost:8080/api/v1/upgradetask/ --header "Content-Type:application/json" -d '{ 
    "upgradeTaskClass":"com.dotmarketing.startup.runonce.Task05195CreatesDestroyActionAndAssignDestroyDefaultActionsToTheSystemWorkflow"
}' | jq
```

And make sure the task were executed properly executing:
`select * from workflow_action_mappings;`

These default actions should be present:
```
NEW
EDIT
PUBLISH
UNPUBLISH
ARCHIVE
UNARCHIVE
DELETE
DESTROY
```

**Comment by jgambarios** [comment]
Related ticket: https://github.com/dotCMS/core/issues/17738

PR: https://github.com/dotCMS/core/pull/17934

**Comment by jgambarios** [comment]
New starter: https://github.com/dotCMS/core/pull/17941

**Comment by alfredo-dotcms** [comment]
@jgambarios all the public pages in general don't load the content at all, in EDIT page in SITE BROWSER, each page container looks empty

**Comment by alfredo-dotcms** [comment]
![image](https://user-images.githubusercontent.com/37185433/74368506-68b13380-4d99-11ea-8482-a9f435c4937e.png)

**Comment by alfredo-dotcms** [comment]
@jgambarios after doing the rollback there are some pages that loads correctly and some that are not:

- Contact Us --> starter.dotcms.com/contact-us
- search functionality --> starter.dotcms.com/search/
- store main page --> starter.dotcms.com/store

**Comment by jgambarios** [comment]
New starter: https://github.com/dotCMS/core/pull/18013

**Comment by bryanboza** [comment]
Fixed, tested on release-5.2.6 // Postgres // FF. New card reported here: #18000
